Headers/GNUstepBase/GSVersionMacros.h

```diff
@@ -66,6 +66,11 @@
 #define MAC_OS_X_VERSION_10_12	1120
 #define MAC_OS_X_VERSION_10_13	1130
 #define MAC_OS_X_VERSION_10_14	1140
+#define MAC_OS_X_VERSION_10_15	101500
```

Comment:
Ahh I just realized that the GS defines are just using 4 digits, whereas on Apple platforms they are using 6 starting with 10.10 (i.e. `101000`). Not sure what’s best to do here.
- I guess editing old defines might cause issues although it might also be better to be in sync with Apple platforms?
- Alternatively we could keep 4 digits up to 10.16 and use 6 digits starting with 11.0?